### DISCUSSION TITLE
S Grades might be too easily obtainable

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#24646] [Q&A] [answered]
Right now grades are only based on accuracy, just like they are for mania on stable. To obtain an S grade all you need is to get 95% accuracy. While this might work well on mania, I believe this might be a bit too easy in other modes, especially osu! where previously you only were able to get an S without misses. In osu!catch an S grade would need 98% or better on stable. 

While I don't have that much experience in other modes, I personally think that at least in osu! only 95% accuracy as a requirement is going to be way too easy to achieve. I would propose to at least change it to 95% acc and no misses.

Interested to hear other peoples opinion on this.

**Comment by Givikap120** [comment]
Slider acc exist, so getting 95% in lazer is harder than in stable
Because of this A rank for 90% seems fair to me, and 95% is exactly between 90% and 100%
Can't say anything about osu!catch

**Comment by LeaPhant** [comment]
Grades should somewhat reflect performances and since aim is an integral part of osu! a miss should be reflected in the grade further than just as a decrease in accuracy. I also consider a play with a miss ('choke') being an A rank as a part of osu!'s identity, so I agree with bringing this behavior back as to not confuse or alienate players.

95% S requirement for osu!catch is definitely too easy, most good plays in catch will be above 99% and anything below 97% means you've barely kept up with the map.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/4184

**Comment by cihe13375** [comment]
I don't agree that S grade should require 0 misses, mainly because it makes getting S grade unfairly difficult on long maps (which are already underhyped for a variety of reasons). If we want to consider # of misses in grades I'd prefer either of the two approaches below (showing the concept only, the exact numbers may be changed):
1) S rank = (>=95% acc) and (<=0.5% miss count). By using the % of misses as criteria, the difficulty of getting S on short and long maps are more balanced;
2) S rank = (>=97.5% acc) or (>=95% acc and 0 misses). It imitates the current mechanism for A and B ranks, and making getting S on long maps not too difficult.

**Comment by ghost** [comment]
I think S ranks lose a bit of meaning if the miss requirement is lax and the acc requirement is only 95% at the same time. It being difficult to get an S rank is what makes it satisfying when you finally get one. A 1 miss on a long map doesn't need an S rank to signify its impressiveness, it's already impressive without muddying what a grade signifies. Also, it's one thing to look at very hard maps and think it's unfair that they're difficult to get, but what about new players? There are very long 2 and 3 star maps as well, should 95% acc alone or even 97% with misses get an S rank on a 3 star map? While it would be nice to standardize everything across all modes, they are basically different games and I don't think it makes sense here. S ranks suggest a nearly perfect score, and what nearly perfect means isn't the same across all modes

**Comment by Rekunan** [comment]
A good solution that could satisfy everyone is adding more ranks, perhaps copying some from other rhythm games.

Here's one proposition I made with some Arcaea inspiration:
<table>
<tr>
 <td>Acc
 <td>Not-FC
 <td>FC
<tr>
 <td>100%
 <td>
 <td>X
<tr>
 <td>99%
 <td>AA+
 <td>SS+
<tr>
 <td>98%
 <td>AA
 <td>SS
<tr>
 <td>95%
 <td>A+
 <td>S+
<tr>
 <td>92%
 <td>A
 <td>S
</table>

The ideas of S=FC, actual values themselves, etc, are all **very experimental**. The main idea I am proposing is **adding more ranks**. My only concern is how hard/time-consuming that would be in terms of development.

**Reply by monochrome22** [reply]
[Here's](https://github.com/ppy/osu/discussions/16998#discussioncomment-2263664) peppy's opinion on changing rank grades. The main issue is confusion.

Personally my only issue with grades is that you can't tell the difference between FC and not FC. Maybe add another color for the combo number to signify FC, like how there is green for perfect. Would need to add combo numbers to profiles then though.

**Reply by Rekunan** [reply]
I had a feeling there was a discussion for that already.

Yeah, I'd also be satisfied if there were an FC marker. I don't think it should be in the combo number, though; in the grade itself, like how Givikap suggested in that discussion, would be good.

**Reply by ghost** [reply]
Who is the "everyone" that we're satisfying? How many people in the community actually wanted a change to grades and didn't like how they work in Stable? When people don't like something, you don't just hear it on github, you hear it while playing multi, on the forums, in YouTube comments... I don't think I've ever heard a complaint about S/SS ranks prior to this discussion, all I've ever heard is that some people feel a sliderbreak should count as a miss and not give an S.

**Reply by Rekunan** [reply]
@lainosu people who prefer acc-based grades, and people wanting a distinction between FC and not-FC.

This also isn't something that isn't fixing what people dislike, as in it's not that people dislike the current system, it's just that they'd probably like this system better.
